client/src/elements/schema-editor/schema-editor-object.js

```diff
@@ -37,6 +39,13 @@ export class SchemaEditorObject {
     this.applyOptions();
   }
 
+  getPointer(propertyName) {
```

Comment:
nice, this wasn't too much work to get it working

Reply:
no, was pretty straightforward.